### PR TITLE
filter out bad pool per request from adam and johnny

### PR DESCRIPTION
**Pull request by MaxMillington**
## What is the purpose of the change:

Filtering out pool 2159

### Linear Task

https://linear.app/osmosis/issue/POL-1363/filter-out-pool-2159

## Brief Changelog

- Filter out bad pool

## Testing and Verifying

This change has been tested locally by rebuilding the website and verified content and links are expected

### CODE REVIEW

File: packages/web/components/complex/pools-table.tsx [hunk 1]
```diff
@@ -36,6 +36,8 @@ import { api, RouterOutputs } from "~/utils/trpc";
 
 import { Tooltip } from "../tooltip";
 
+const FILTERABLE_IDS = ["2159"];
```
Comment (jonator) [inline packages/web/components/complex/pools-table.tsx:39]: Perhaps we can add a ternary operator that creates an array for testnet & mainnet.

Like:
`const FILTERABLE_IDS = IS_TESTNET ? [] : ["2159"];`

**Comment by jonator** [comment]
I commented in slack, but I'm also no longer seeing the Cosmo pool in prod

File: packages/web/components/complex/pools-table.tsx [hunk 2]
```diff
@@ -176,10 +178,14 @@ export const PoolsTable = (props: PropsWithChildren<PoolsTableProps>) => {
...
-    [poolsPagesData]
-  );
+  const poolsData = useMemo(() => {
+    const allItems =
+      poolsPagesData?.pages.flatMap((page) => {
+        return page?.items.filter((item) => !FILTERABLE_IDS.includes(item.id)); // Filter out ids in FILTERABLE_IDS
+      }) ?? [];
+
+    return allItems;
+  }, [poolsPagesData]);
```
Comment (JoseRFelix) [inline packages/web/components/complex/pools-table.tsx:188]: Do we only want to filter this pool in the pools table? If not, I suggest modifying https://github.com/osmosis-labs/osmosis-frontend/blob/288c023d8100418b8f5a2b607f532dd6e0014112/packages/server/src/queries/complex/pools/index.ts#L89 instead, so the changes propagate to all components that rely on pools.

**Review by JoseRFelix — APPROVED** [review]

**Comment by raphaellafar** [comment]
I reiterate the request to unblock liquidity pool 2159. As confirmed in the thread https://forum.osmosis.zone/t/dao-proposal-establish-transparent-rules-for-token-and-liquidity-pool-blocking/3233 , this pool should not be blocked.

**Pull request merged**